CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ HumHub Changelog
 - Fix #5066: Fix pin and archive global content without container
 - Fix #5107: Fix captcha client validation after request new code
 - Fix #22: Fix misplaced OEmbed provider buttons on general settings form
+- Fix #5128: Prerequisites: Increase PHP Min version to 7.3
```

Comment:
@yurabakhtin Please use the `develop` branch and add it to CHANGELOG_DEV  for `beta.2`

Reply:
@luke- I changed the base of this PR from `master` to `develop` but it didn't give me to edit files from the `develop` branch, so I had to create a new PR https://github.com/humhub/humhub/pull/5133 instead of this, and this PR should be deleted.
PS: It would be good to know what branch to use as base before I start a new task, because currently I don't know a way to switch a base without creating new branch and PR, if you know please tell me, thanks.

Reply:
@yurabakhtin Ok, I try to add the target branch to the issues if unclear.